examples/php/01_basic.re

```diff
@@ -0,0 +1,16 @@
+<?php
+// re2php $INPUT -o $OUTPUT
```

Comment:
I think it should always be the first line of the file (as expected by run_tests.py).

Reply:
Impossible, as `<?php` must be on 1st line.

Reply:
I see. Then I suggest adding some sed magic in `examples/php/__run_all.sh` that will cut off the comments in the temporary file (this one and the one autogenerated by re2php).